packages/cli/lib/projects.js

```diff
@@ -65,82 +71,76 @@ const getProjectConfig = async projectPath => {
   });
 
   if (!configPath) {
-    return null;
+    return { projectConfig: null, projectDir: null };
   }
 
   try {
-    const projectConfig = fs.readFileSync(configPath);
-    return JSON.parse(projectConfig);
+    const config = fs.readFileSync(configPath);
+    const projectConfig = JSON.parse(config);
+    return {
+      projectDir: path.dirname(configPath),
+      projectConfig,
+    };
   } catch (e) {
     logger.error('Could not read from project config');
   }
 };
 
-const createProjectConfig = async (projectPath, projectName) => {
-  const projectConfig = await getProjectConfig(projectPath);
-  const projectConfigPath = path.join(projectPath, 'hsproject.json');
+const createProjectConfig = async (projectPath, projectName, template) => {
```

Comment:
Could you elaborate on the different cases being handled here? It's a little tricky to follow

Reply:
I'm actually going to rework this now since we're going to allow developers to move forward even if a project exists so I'll make sure this is clearer.

There are 3 cases here:
- Project already exists at `--location`
- Project exists in a parent of `--location`
- Project exists in a child folder of `--location`

The last "child folder" check can be pretty slow if there are a lot of files and since we're allowing devs to override, I'm thinking it make sense to remove this case.

Reply:
Took a pass at updating this FYI